xio-core/src/main/java/com/xjeffrose/xio/config/DynamicClientConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.xjeffrose.xio.config;
+
+import java.util.Objects;
+
+/** This class is a POJO representation of the input data that maps to a ClientConfig */
+public class DynamicClientConfig {
+  private String clientName;
+  private String ipAddress;
+  private int port;
+  private boolean tlsEnabled;
+
+  public DynamicClientConfig(String clientName, String ipAddress, int port, boolean tlsEnabled) {
+    this.clientName = clientName;
+    this.ipAddress = ipAddress;
+    this.port = port;
+    this.tlsEnabled = tlsEnabled;
+  }
+
+  public String getClientName() {
+    return clientName;
+  }
+
+  public String getIpAddress() {
+    return ipAddress;
+  }
+
+  public int getPort() {
+    return port;
+  }
+
+  public boolean isTlsEnabled() {
+    return tlsEnabled;
+  }
+
+  public boolean equals(Object other) {
```

Comment:
There's a Lombok helper for equals and hashcode. Can put `@EqualsAndHashCode` above the class name

Reply:
ok applied that